pages/3_Compare_Cases.py

```diff
@@ -1,26 +1,79 @@
 # pylint: disable=invalid-name
 
+import extra_streamlit_components as stx
 import streamlit as st
 
 from sibylapp.compute.context import get_term
 from sibylapp.view import entity_difference
 from sibylapp.view.utils import filtering, setup
 
-setup.setup_page()
+setup.setup_page(return_row_ids=True)
+
+
+tab = stx.tab_bar(
+    data=[
+        stx.TabBarItemData(
+            id=1, title=f"Difference within single {get_term('entity')}", description=""
+        ),
+        stx.TabBarItemData(
+            id=2, title=f"Difference between {get_term('entity', plural=True)}", description=""
+        ),
+    ],
+    default=1,
+)
+
 
 # Sidebar ------------------------------------
-filtering.view_entity_select(eid_text="eid", prefix="first", default=0)
-filtering.view_entity_select(eid_text="eid_comp", prefix="second", default=1)
-entity_difference.view_instructions()
+if tab == "1":
+    filtering.view_entity_select(add_prediction=False)
+    eid = st.session_state["eid"]
+    row_ids = st.session_state["row_id_dict"][eid]
+    if len(row_ids) < 2:
+        st.subheader(
+            "This entity contains one or zero rows of data. Difference within this entity not"
```

Comment:
I think in the future, we should make a configuration option that prevents the "within entity" tab from appearing if its not relevant (but we can do that in another PR)

Reply:
For now, please update the entity to be `get_term('Entity', lower=True)` in the error message. I'd also recommend using `st.warning()` instead of `st.header()` for a better look